engine/runtime/src/main/java/org/enso/interpreter/runtime/callable/atom/EmptyLayout.java

```diff
@@ -0,0 +1,3 @@
+package org.enso.interpreter.runtime.callable.atom;
+
+public class EmptyLayout {}
```

Comment:
A left-over from some previous attempts? I don't see it being used anywhere.

Reply:
Right, good catch